backend/pps/sch.py

```diff
@@ -69,20 +69,20 @@ def total_potency_spreadsheet_port(self, character_stats, caster_tax, ed_per_min
 
     @classmethod
     def get_cycle(cls, character_stats, caster_tax):
-        '''
+        """
         Actual time taken by a 180s rotation, is lower than 180s
-        '''
+        """
         short_gcd = character_stats.get_gcd()
         result = 0
         # 1 bio + x Broils and 4 R2s/3min
         if (30 - 2 * short_gcd) % (short_gcd + caster_tax) > 1.5 :
             result += 6 * (
```

Comment:
i still think there is something really cursed about this line breaking but i'm just gonna let it go tbh...